homeassistant/components/aladdin_connect/coordinator.py

```diff
@@ -41,4 +41,18 @@ def __init__(
     async def _async_update_data(self) -> GarageDoor:
         """Fetch data from the Aladdin Connect API."""
         await self.client.update_door(self.data.device_id, self.data.door_number)
+        status, battery_level = (
+            self.client.get_door_status(self.data.device_id, self.data.door_number),
+            self.client.get_battery_status(self.data.device_id, self.data.door_number),
+        )
+        if status is None:
```

Comment:
Why not write it as?

```suggestion
        if (status := self.client.get_door_status(self.data.device_id, self.data.door_number)) is None:
```